templates/search/search_aggregates_v8/query.sql

```diff
@@ -16,16 +20,19 @@ SELECT
   SUM(search_with_ads) as search_with_ads,
   SUM(unknown) as unknown
 FROM
-  search_clients_daily_v8
+  search_clients_daily_v8_bq_test
 WHERE
   submission_date = @submission_date
 GROUP BY
   submission_date,
+  addon_version,
   app_version,
   country,
   distribution_id,
+  engine,
   locale,
+  os,
+  os_version,
```

Comment:
these should move to the end of the `GROUP BY` list, to keep consistent with the column order